assets/src/styles/blocks/ENForm/components/_enform-side-style.scss

```diff
@@ -70,6 +70,10 @@
     margin-top: 0;
     margin-bottom: 0;
 
+    margin-left: calc(((100vw - 100%) / 2) * -1);
+    padding-left: calc(((100vw - 100%) / 2));
+    padding-right: calc(((100vw - 100%) / 2));
+
```

Comment:
What's the motivation of these `calcs`? To make it wider? If yes, you can add this [class](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/02041739f599bea14757770d14c00eb1205123ae/assets/src/styles/blocks/WideBlocks.scss#L1-L10)

Reply:
It's replacing that code https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/master/public/js/enform_side_style.js  (which I forgot to remove apparently, doing that now :grimacing:), the .block-wide class is missing the paddings used to narrow the form on width < 992px